src/SDK/SmartDetectors.Extensions/Extensions/StringExtensions.cs

```diff
@@ -6,18 +6,21 @@
 
 namespace Microsoft.Azure.Monitoring.SmartDetectors.Extensions
 {
+    using System.Collections.Generic;
```

Comment:
>using [](start = 4, length = 5)

Please update the extensions nuproj with newer version.
After the PR is complete please publish the nuget.

Reply:
Will be published as part of Eran's changes

---
In reply to: [231027754](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/18#discussion_r231027754) [](ancestors = 231027754)